mackup/applications/synapse.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+# http://synapse.zeitgeist-project.com/wiki/index.php?title=Main_Page
+name = Synapse
+
+[configuration_files]
+.config/synapse
```

Comment:
It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).